composer.json

```diff
@@ -6,7 +6,7 @@
     "homepage": "https://github.com/wp-media/wp-rocket-cli",
     "license": "MIT",
     "require": {
-        "php": ">=5.3.0"
+        "php": ">=5.4"
```

Comment:
we should probably increase this to 5.6, what do you think?

Reply:
till now our code doesn't need 5.6, I only updated it to 5.4 to support type hint on methods arguments with default value.
If you need me to update it to be ready to be used on the following updates, it would be better for sure.

Reply:
Let's update it to require 5.6